composer.json

```diff
@@ -1,20 +1,24 @@
 {
-    "name": "paquettg/php-html-parser",
+    "name": "thesoftwarefanatics/php-html-parser",
     "type": "library",
-    "version": "1.7.0",
     "description": "An HTML DOM parser. It allows you to manipulate HTML. Find tags on an HTML page with selectors just like jQuery.",
     "keywords": ["html", "dom", "parser"],
-    "homepage": "https://github.com/paquettg/php-html-parser",
+    "homepage": "https://github.com/thesoftwarefanatics/php-html-parser",
     "license": "MIT",
     "authors": [
         {
             "name": "Gilles Paquette",
             "email": "paquettg@gmail.com",
             "homepage": "http://gillespaquette.ca"
+        },
+        {
+            "name": "The Software Fanatics GmbH",
+            "email": "team@thesoftwarefanatics.com",
```

Comment:
This email address is not available from outside the organisation. I'd rather use `dev@...`.

Reply:
Whoops 😉